tests/unit/htdocs/class/cache/XoopsCacheTest.php

```
@@ -0,0 +1,686 @@
+
+        $this->assertTrue($result, 'Garbage collection should return true');
+    }
+
+    public function testGarbageCollectionWithoutInitializedEngine()
+    {
+        // Create a new instance to avoid initialized engines
+        $result = @$this->cache->gc();
+
+        // Should return false without initialized engine
+        $this->assertFalse($result);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 398. (undefined)

(ErrorControlOperator)